### DISCUSSION TITLE
噴嚏打不停？靠1招就能止住！

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1766] [General]
「嗨啾！」每當換季時，天氣驟變會讓很多人開始過敏打噴嚏，讓有過敏體質的人相當困擾，有時一過敏起來，整天的工作效率都大打折扣，但你有聽過打噴嚏是有用方法可以停止的嗎？

下次當你想要打噴嚏時，迅速用手按住「上巴」的位置（嘴唇上方，鼻孔下方），你會發現想打噴嚏的感覺瞬間消失。
其原理是，打噴嚏是因為「三叉神經」傳導致大腦，要排出鼻黏膜裡的東西，導致我們有想打噴嚏的感覺，按住上巴是因為給予大腦「新的刺激」，來達到抑止打噴嚏的效果。下次如果想打噴嚏，不妨用這招試試看！

資料來源：https://www.storm.mg/lifestyle/764909?mode=whole

對於換季過敏兒來說，打噴嚏真的很煩！還會擦到鼻子破皮
我自己試過這個方法，還滿有效的，不過有時候都會忘記做這個動作

**Comment by mrweixd** [comment]
學會了，那打嗝止不住的話該怎麼辦？

**Reply by 111426027** [reply]
如果我一直打嗝的話，會按住右手中指的最上節，同時憋氣十秒，就不會打嗝了。另一種方法是很多水也有用!

**Comment by davidteng00000** [comment]
這招是個不錯的小訣竅，按住「上巴」位置確實能緩解噴嚏。但打嗝則可以按住右手中指最上節，同時憋氣十秒，或者多喝水也有幫助！

**Comment by poiu6fi** [comment]
學到了!!下次過敏的時候馬上是看看，感謝分享
